core/domain/prod_validation_jobs_one_off.py

```diff
@@ -3008,6 +3010,91 @@ def _get_external_id_relationships(cls, item):
                 'exp_ids', exp_models.ExplorationModel, [item.id])]
 
 
+class ExplorationMathRichTextInfoModelValidator(BaseModelValidator):
+    """Class for validating ExplorationMathRichTextInfoModel."""
+
+    @classmethod
+    def _get_external_id_relationships(cls, item):
+        return [
+            ExternalModelFetcherDetails(
+                'exp_ids', exp_models.ExplorationModel, [item.id])]
+
+    @classmethod
+    def _validate_latex_values(
+            cls, item, field_name_to_external_model_references):
+        """Validate that question_content model is equal to
```

Comment:
What does this have to do with anything in this method?

**Please proofread your PR and make sure all your docstrings are correct.** Treat correctness of docstrings with the same seriousness as you treat correctness of code, otherwise it can be very misleading for other developers.

Reply:
done